UIKitHomework/UIKitHomework/FoodViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  FoodViewController.swift
+//  UIKitHomework
+//
+//  Created by Vera Zaitseva on 30.06.2021.
+//
+
+import UIKit
+
+class FoodViewController: UIViewController {
```

Comment:
```suggestion
final class FoodViewController: UIViewController {
```
все классы ниже от которых нет наследования должны быть помечены final